device_db.yaml

```diff
@@ -842,6 +842,32 @@ BSEED_TOUCH_TS0003_2:
   info: Backlight on D4i
   threads: https://github.com/romasku/tuya-zigbee-switch/issues/240
   store: https://www.aliexpress.com/item/1005004187227246.html
+BSEED_TOUCH_2_TS0013:
+  human_name: BSEED 3-gang touch switch
+  category: switch
+  power: mains
+  neutral: without
+  device_type: end_device
+  tuya_model_name: TS0013
+  tuya_manufacturer_name: _TZ3000_kfwhmnvc
+  stock_converter_manufacturer: Tuya
+  stock_converter_model: TS0013
+  override_z2m_device: null
+  tuya_module: ZTU
+  mcu_family: Telink
+  mcu: TLSR8258
+  config_str: kfwhmnvc;TS0013-2-BS;LC3;SB5u;RC0B6;ID2;SB4u;RA1D7;IC2;SD4u;RA0C1;ID3;M;SLP;
+  alt_config_str: null
+  old_manufacturer_names: null
+  old_zb_models: null
+  tuya_manufacturer_id: 4417
+  tuya_image_type: 54179
+  firmware_image_type: 43580
+  build: yes
+  status: fully_supported
+  info: Supported
```

Comment:
I wasn't sure if we want to mention something here about bi-stable relays.

Reply:
If you installed the firmware and confirmed it's working with the given pinout (including SLP) - there's nothing to mention. 🙂 Thank you very much!